crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -29,6 +29,8 @@ class CrowdfundingParticipant(models.Model):
     invoice_line_ids = fields.One2many(
         "account.invoice.line", "crowdfunding_participant_id", string="Donations"
     )
+    campaign_id = fields.Many2one('utm.campaign', 'campaign_id',
```

Comment:
You forgot the _inherits part. Please be more rigurous when doing your work. It's a pain to always must double check your work.

Reply:
Sorry I was not concentrated doing this change